ci-operator/step-registry/ibmcloud/provision/vpc/endpoints-gateway/ibmcloud-provision-vpc-endpoints-gateway-ref.yaml

```diff
@@ -42,5 +45,25 @@ ref:
     default: ""
     documentation: |-
       GlobalTagging endpoint
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+    documentation: |-
+      COSConfig endpoint
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+    documentation: |-
+      GlobalCatalog endpoint
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+    documentation: |-
+      KeyProtect endpoint
+  - name: SERVICE_ENDPOINT_CIS
+    default: ""
+    documentation: |-
+      CIS endpoint, which is used in External cluster.
+  - name: SERVICE_ENDPOINT_HyperProtect
```

Comment:
I did not see the default endpoint for HyperProtect defined in `eps_default.json`, if we are not ready to test it, pls remove it.

Reply:
I will add the following in  eps_default.json, but considering the costing issue (from dev), I can not test now
  "HyperProtect": "https://api.private.${REGION}.hs-crypto.cloud.ibm.com"

Reply:
okay